app.rb

```diff
@@ -40,63 +40,62 @@
 helpers do
   def protected!
     return if authorized?
-    headers['WWW-Authenticate'] = 'Basic realm="Restricted Area"'
+    headers["WWW-Authenticate"] = 'Basic realm="Restricted Area"'
     halt 401, "Not authorized\n"
   end
 
   def authorized?
-    @auth ||=  Rack::Auth::Basic::Request.new(request.env)
-    @auth.provided? && @auth.basic? && @auth.credentials == [ENV.fetch('HTTP_BASIC_USER'), ENV.fetch('HTTP_BASIC_PASSWORD')]
+    @auth ||= Rack::Auth::Basic::Request.new(request.env)
+    @auth.provided? && @auth.basic? && @auth.credentials == [ENV.fetch("HTTP_BASIC_USER"), ENV.fetch("HTTP_BASIC_PASSWORD")]
   end
 end
 
-before { protected! unless request.path_info == '/check' }
+before { protected! unless request.path_info == "/check" }
 
-get '/' do
-  redirect('/board/dxw')
+get "/" do
+  redirect("/board/dxw")
 end
 
-get '/hoxton' do
-  redirect('/board/hoxton')
+get "/hoxton" do
+  redirect("/board/hoxton")
 end
 
-get '/leeds' do
-  redirect('/board/leeds')
+get "/leeds" do
+  redirect("/board/leeds")
 end
 
-get '/board/:slug' do
-  board = BOARDS[params['slug'].to_sym]
+get "/board/:slug" do
+  board = BOARDS[params["slug"].to_sym]
   @board_name = board[:name]
   @show_clock = board[:show_clock]
   @rooms = board[:rooms]
   @today = Date.today
   haml :multi_room
 end
 
-get '/room/:slug.json' do
+get "/room/:slug.json" do
   content_type :json
 
-  room = ROOMS[params['slug'].to_sym]
+  room = ROOMS[params["slug"].to_sym]
 
   {
-    :colour => room.presence_colour_rgb,
-    :enable_presence_device => DateTime.now.hour.between?(PRESENCE_INDICATORS_ACTIVE_START, PRESENCE_INDICATORS_ACTIVE_END),
-    :empty => room.empty,
-    :minutes_to_next_event => room.minutes_to_next_event,
-    :minutes_to_end_of_event => room.minutes_to_end_of_event,
-    :upcoming_event_today => room.upcoming_event_today,
-    :events => (room.events unless params['compact'])
+    colour: room.presence_colour_rgb,
+    enable_presence_device: DateTime.now.hour.between?(PRESENCE_INDICATORS_ACTIVE_START, PRESENCE_INDICATORS_ACTIVE_END),
+    empty: room.empty,
+    minutes_to_next_event: room.minutes_to_next_event,
+    minutes_to_end_of_event: room.minutes_to_end_of_event,
+    upcoming_event_today: room.upcoming_event_today,
+    events: (room.events unless params["compact"])
   }.compact.to_json
 end
 
-get '/room/:slug' do
-
-  @room = ROOMS[params['slug'].to_sym]
+get "/room/:slug" do
+  @room = ROOMS[params["slug"].to_sym]
   @today = Date.today
 
   haml :room
 end
 
-get '/check' do
-  'Im alive!'
+get "/check" do
+  "Im alive!"
```

Comment:
On the other hand, having double quotes allows us to fix this longstanding eyesore 😼